src/garbage.rs

```diff
@@ -29,11 +29,13 @@ use arrayvec::ArrayVec;
 use deferred::Deferred;
 
 /// Maximum number of objects a bag can contain.
-#[cfg(not(feature = "strict_gc"))]
-const MAX_OBJECTS: usize = 64;
-#[cfg(feature = "strict_gc")]
-const MAX_OBJECTS: usize = 4;
-
+cfg_if! {
+    if #[cfg(feature = "sanitize")] {
+        const MAX_OBJECTS: usize = 4;
+    } else {
```

Comment:
This removes the `strict_gc` feature, shouldn't it be `feature = sanitize` OR `feature = strict_gc` instead?

Reply:
I think we're okay with the single feature `sanitize` for now. Let's add `strict_gc` (or maybe call it `aggressive_gc`?) alongside it later when we find a use for it.